modules/consensus/maintenance.go

```diff
@@ -31,26 +30,37 @@ func (cs *ConsensusSet) applyMinerPayouts(tx *bolt.Tx, pb *processedBlock) error
 			MaturityHeight: pb.Height + types.MaturityDelay,
 		}
 		pb.DelayedSiacoinOutputDiffs = append(pb.DelayedSiacoinOutputDiffs, dscod)
-		err := cs.commitTxDelayedSiacoinOutputDiff(tx, dscod, modules.DiffApply)
-		if err != nil {
-			return err
-		}
+		commitDelayedSiacoinOutputDiff(tx, dscod, modules.DiffApply)
 	}
-	return nil
 }
 
 // applyMaturedSiacoinOutputs goes through the list of siacoin outputs that
 // have matured and adds them to the consensus set. This also updates the block
 // node diff set.
-func (cs *ConsensusSet) applyMaturedSiacoinOutputs(tx *bolt.Tx, pb *processedBlock) error {
+func applyMaturedSiacoinOutputs(tx *bolt.Tx, pb *processedBlock) {
 	// Skip this step if the blockchain is not old enough to have maturing
 	// outputs.
-	if !(pb.Height > types.MaturityDelay) {
-		return nil
+	if !(pb.Height >= types.MaturityDelay) {
```

Comment:
I'm not sure if it applies in go, but in some cases if either side is NaN or null the >= would be false